app/src/ui/branches/pull-request-list-item.tsx

```diff
@@ -27,8 +27,8 @@ export class PullRequestListItem extends React.Component<
   public render() {
     const timeAgo = moment(this.props.created).fromNow()
     const title = this.props.title
-    const subtitle = `#${this.props.number} opened ${timeAgo} by ${this.props
-      .author}`
+    const author = this.props.author
```

Comment:
`const { author, number } = this.props`?

Reply:
Good point, I always forget that's a thing!